src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1364,4 +1366,16 @@ public static void sendRecipeBook(NetHandlerPlayServer connection, State state,
         if (recipes.size() > 0 || display.size() > 0)
             connection.sendPacket(new SPacketRecipeBook(state, recipes, display, isGuiOpen, isFilteringCraftable));
     }
+    
+    /**
+     * Return if the passed Player currently has the ability to use elytra flight
+     * @param entity
+     * @return
+     */
+    public static boolean hasElytraFlightAbility(EntityLivingBase player, ItemStack itemstack) {
```

Comment:
Please follow the formatting used elsewhere in the project. Braces go on a new line, spaces after `if` before the parenthesis, spaces around comparison operators, etc.